native/src/models/__tests__/ContentLoadCriterion.spec.ts

```diff
@@ -0,0 +1,21 @@
+import moment from 'moment'
+
+import { ContentLoadCriterion } from '../ContentLoadCriterion'
+
+describe('ContentLoadCriterion', () => {
+  const stateView = new ContentLoadCriterion(
+    {
+      forceUpdate: false,
+      shouldRefreshResources: false
+    },
+    false
+  )
+  it('should not update content if last update was now', () => {
+    const now = moment.utc()
+    expect(stateView.shouldUpdate(now)).toBeFalsy()
+  })
+  it('should update if content was updated 5 minutes before start of today', () => {
+    const beforeStartDay = moment.utc().startOf('day').subtract(5, 'minutes')
+    expect(stateView.shouldUpdate(beforeStartDay)).toBeTruthy()
+  })
```

Comment:
Maybe you could add one more test with the date set to 5 minutes after midnight?
I think you should also mock the current date, otherwise this test will fail if executed between 0:00 and 0:05. You could try the following to do so: https://github.com/facebook/jest/issues/2234#issuecomment-730037781

Reply:
@klinzo 
updated tests and mocked current system time